### PR TITLE
Update Min Dong Chinese language

### PR DESCRIPTION
**Pull request by yjfvictor**
Requested at https://phabricator.wikimedia.org/T139010

### CODE REVIEW

File: data/langdb.yaml [hunk 1]
```diff
@@ -627,6 +629,7 @@ languages:
   zh-sg: [Hans, [AS], 中文（新加坡）]
   zh-tw: [Hant, [AS], 中文（台灣）]
   zh-yue: [yue]
+  zh-cdo: [cdo]
```
Comment (Nikerabbit) [inline data/langdb.yaml]: `zh-yue` exists for compatibility due to legacy use of that code. `zh-cdo` does not have legacy use so I think it should not be added.

Comment (yjfvictor) [reply data/langdb.yaml]: OK, I have removed zh-cdo.

**Comment by MarcoAurelioWM** [comment]
data/langdb.yaml needs rebasing

**Comment by amire80** [comment]
A new version here: https://github.com/wikimedia/language-data/pull/127

**Pull request closed without merging**